local.py

```diff
@@ -0,0 +1,23 @@
+import os
```

Comment:
from the name _local.py_ I don't really know what the file is doing. Maybe _main.py_ or _run.py_ is more informative?

Reply:
In my understanding, this file is more like a test (the 'main' way to run this library is via server.py).

So I don't like main and run.py conflicts 'run'... but I'm open to any other name. I added a small doc to the top of the file.

Reply:
The doc is helpful, thanks! Then let's keep the name.